### PR TITLE
Rename Token V2 to Digital Asset (DA)

### PR DESCRIPTION
**Pull request by saharct**

### CODE REVIEW

**Review by 0xmaayan — APPROVED** [review]

File: developer-docs-site/docs/reference/glossary.md
Comment (xbtmatt) [inline developer-docs-site/docs/reference/glossary.md:201]:
```suggestion
- For the legacy Aptos Token Standard (aptos_token::token), a **fungible token** is a token that is interchangeable with other identical tokens (i.e., tokens that share the same `TokenId`). This means the tokens have the same `creator address`, `collection name`, `token name`, and `property version`.
```

Should we refer to it as legacy here to be consistent? It's also much more clear

Comment (saharct) [reply developer-docs-site/docs/reference/glossary.md:201]: Yeah, let's maintain consistency. Thanks!

File: developer-docs-site/docs/standards/digital-asset.md
Comment (xbtmatt) [inline developer-docs-site/docs/standards/digital-asset.md:88]:
```suggestion
All Digital Asset modules are deployed at the address `0x4`.
```

Maybe this? Or "reserved framework address" or something. Just to be a little clearer, some people might not know we have framework modules here and `0x4` doesn't look like an address to someone new

Comment (saharct) [reply developer-docs-site/docs/standards/digital-asset.md:88]: Good point. Let's go with "reserved framework address".

File: developer-docs-site/docs/standards/fungible-asset.md
Comment (xbtmatt) [inline developer-docs-site/docs/standards/fungible-asset.md:10]:
```suggestion
A Fungible Asset (FA) is a core framework component within Aptos that enables the tokenization of various assets, including commodities, real estate, and financial instruments, and facilitate the creation of decentralized financial applications.
```

Reads a little odd otherwise

File: developer-docs-site/docs/standards/digital-asset.md
Comment (xbtmatt) [inline developer-docs-site/docs/standards/digital-asset.md:348]:
```suggestion
This can be easily built by creating an asset that is both a Digital Asset (DA) and a Fungible Asset (FA), resulting in a Digital and Fungible Asset (DFA). After the creator creates the Armor collection and the
```

Clarifying a bit here, it's a bit awkward otherwise. Don't mean to nit, just think it could read a little smoother

**Review by xbtmatt — APPROVED** [review]
I disabled auto merge in case you wanna change anything before merging. LGTM other than the little stuff

File: developer-docs-site/docs/standards/fungible-asset.md
Comment (saharct) [reply developer-docs-site/docs/standards/fungible-asset.md:10]: @xbtmatt it was meant to be a shortening of "The Aptos Fungible Asset Standard is a core..."
Adding "a" makes it read like we're talking about an individual instance of FA.

Comment (xbtmatt) [reply developer-docs-site/docs/standards/fungible-asset.md:10]: Yea I wasn't sure how to write it exactly, I think the long form you mentioned is clearest but I'll defer to you on that :p

**Pull request merged**
